.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       matrix:
         # "macos-latest" removed for now as Docker action is slow
         os: [ "ubuntu-latest" ]  # TODO: add "windows-latest"
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12-dev"]
```

Comment:
not sure if we can support 3.11+ 
@nenb WDYT?

Reply:
Hi @thejumpman2323 , you can approval the CI run and then it will test whether it work in 3.11

Reply:
done :)

Reply:
@zhongjiajie 
we have decided to only add 3.11 in the CI,
can you remove 3.12-dev from this pr 
and we can get this pr merged!

Thanks